### ISSUE TITLE
MapBufferRange

### ISSUE THREAD

**Issue by github-manu** [danginsburg/opengles3-book#6]
Book, page 156, glMapBufferRange()
There are two mistakes:
- GL_MAP_INVALIDATE_BUFFER_BIT: "This flag can only be used in combination with GL_MAP_READ_BIT" instead of "This ﬂag may not be used in combination with MAP_READ_BIT" (from the spec ES 3.1, page 55)
- GL_MAP_FLUSH_EXPLICIT_BIT: "This flag cannot be used in combination with GL_MAP_WRITE_BIT" instead of "This ﬂag may only be used in conjunction with MAP_WRITE_BIT." (from the spec ES 3.1, page 55)

**Comment by ndhb** [comment]
I think the "glUnmapBuffer" also says "Only target GL_ARRAY_BUFFER is allowed" where you wonder it must be a typo (all 8 targets must be unmappable if they are mappable).

**Comment by danginsburg** [comment]
Indeed, glUnmapBuffer should accept the same targets as glMapBufferRange.
